PolyDeploy/PolyDeploy.dnn

```diff
@@ -66,31 +66,31 @@
 
         <component type="Script">
           <scripts>
-
+            <basePath>DesktopModules/Cantarus/PolyDeploy/</basePath>
```

Comment:
I can only seem to get the installation to succeed if I change this value to `DesktopModules\Cantarus\PolyDeploy\`.

Reply:
Please can you remove the trailing slash from this path.